src/sentry/integrations/base.py

```diff
@@ -308,8 +317,11 @@ def get_client(self) -> Any:
         # Return the api client for a given provider
         raise NotImplementedError
 
-    def get_default_identity(self) -> Identity:
+    def get_default_identity(self) -> Identity | None:
         """For Integrations that rely solely on user auth for authentication."""
+
+        if not self.org_integration:
```

Comment:
Hmm, did the previous implementation raise `NotFound`?  Should we be instead raising `Identity.NotFound` in this case?

Reply:
Good call, fixed